agents/mongodb/internal/profiler/monitors.go

```diff
@@ -85,10 +84,10 @@ func (ms *monitors) MonitorAll() error {
 			dbName,
 		)
 		// ... and start it
-		err := m.Start()
-		if err != nil {
-			log.Println(err)
-			return err
+		e := m.Start()
+		if e != nil {
+			ms.logger.Debugf("couldn't start monitor, reason: %v", e)
+			break
```

Comment:
Please do not change logic, it is too late now.

Reply:
It could help to see at least something when an error happens... :(